fuzzing/replay/status_util.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Utilities for working with absl::Status values.
+
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
+
+namespace fuzzing {
+
+// Creates an error status value that includes the given `message` and a
+// representation of the `errno_value`.
```

Comment:
nitnit: `representation` sounds a bit abstract, how about `description`?

also I feel that `errno` is enough instead of `errno_value`, since `errno` already implies that it is a number - leaving it to you.

Reply:
> nitnit: `representation` sounds a bit abstract, how about `description`?

Done.
 
> also I feel that `errno` is enough instead of `errno_value`, since `errno` already implies that it is a number - leaving it to you.

I agree, but it turns out that `errno` is defined as a macro, at least on the glibc version on my machine, so when I try to use it as a variable name, I get weird compilation errors.

After I ended up using `errno_value`, I saw that POSIX man pages use `errnum` to refer to this value in arguments, but I didn't end up making the change here. If you think it makes a difference, I can revisit this.